.github/workflows/enos-run.yml

```diff
@@ -155,20 +155,10 @@ jobs:
           name: test-e2e-output.zip
           path: enos/test*.out
           retention-days: 5
-      - name: Retry Enos scenario
-        id: run_retry
-        if: steps.run.outcome == 'failure'
-        env:
-          ENOS_VAR_aws_region: us-east-1
-          ENOS_VAR_aws_ssh_keypair_name: enos-ci-ssh-key
-          ENOS_VAR_aws_ssh_private_key_path: ./support/private_key.pem
-          ENOS_VAR_local_boundary_dir: ./support/
-          ENOS_VAR_crt_bundle_path: ./support/boundary.zip
-          ENOS_VAR_tfc_api_token: ${{ secrets.TF_API_TOKEN }}
-          ENOS_VAR_test_email: ${{ secrets.SERVICE_USER_EMAIL }}
-        run: |
-          export ENOS_VAR_enos_user=$GITHUB_ACTOR && \
-          enos scenario run --timeout 60m0s --chdir ./enos ${{ matrix.filter }}
```

Comment:
Did you mean to include this removal of the retry?

Reply:
> Most enos-side failures I've seen don't get resolved unless you do a destroy first

In cases where the retry is working, a failure won't be reported. So you might not see it. You could add another slack notification to notify when a retry is going to happen. Then you would have visability into how often a retry is happening, and how much it is "working".

Reply:
I'll put the retry back in. I'll set it so the notifications send anytime the first attempt fails and can tell from there if a retry fixes it or not.